src/build_cfg.c

```diff
@@ -393,6 +393,22 @@ static const Variable *build_struct_init(struct State *st, const Type *type, con
     return instance;
 }
 
+const Variable *make_zero(struct State *st, const Type *t, Location location)
+{
+    assert(is_integer_type(t));
+    Constant c = {
+        .kind = CONSTANT_INTEGER,
+        .data.integer = {
+            .width_in_bits = t->data.width_in_bits,
+            .is_signed = (t->kind == TYPE_SIGNED_INTEGER),
+            .value = 0,
+        },
+    };
```

Comment:
This kind of code appears in many places. Maybe it's time to make it a function.